src/main/java/shortestpath/PathTileOverlay.java

```diff
@@ -182,16 +177,16 @@ private void drawTile(Graphics2D graphics, WorldPoint point, Color color, int co
 
         if (counter >= 0 && !TileCounter.DISABLED.equals(config.showTileCounter())) {
             if (TileCounter.REMAINING.equals(config.showTileCounter())) {
-                counter = (!plugin.currentPath.loading ?
-                    plugin.currentPath.getPath().size() : plugin.currentPath.currentBest().size()) - counter - 1;
+                counter = (!plugin.currentPath.isDone() ?
+                        plugin.currentPath.getPath().points.size() : plugin.currentPath.getPath().points.size() - counter - 1);
             }
             String counterText = Integer.toString(counter);
             graphics.setColor(Color.WHITE);
             graphics.drawString(
-                counterText,
-                (int) (poly.getBounds().getCenterX() -
-                    graphics.getFontMetrics().getStringBounds(counterText, graphics).getWidth() / 2),
-                (int) poly.getBounds().getCenterY());
+                    counterText,
+                    (int) (poly.getBounds().getCenterX() -
+                            graphics.getFontMetrics().getStringBounds(counterText, graphics).getWidth() / 2),
+                    (int) poly.getBounds().getCenterY());
```

Comment:
Your formatting changes applied to untouched code too. Please revert this if possible, as it is unnecessary for the RuneLite developers to re-review this.